### ISSUE TITLE
Is cuda/any gpu needed

### ISSUE THREAD

**Issue by Rhysjosmin** [vcg-uvic/viper#10]
Do i need to have cuda or any other gpu to run this or can i do it only with a cpu?

**Comment by drebain** [comment]
Yes, the solver implementation here is set up to require CUDA, as the simulation would probably not be real-time on a CPU. That  said, the code uses Thrust to launch the GPU kernels, and could probably be modified to run on CPU.
